### PR TITLE
Update documentation of user controls

### PR DESCRIPTION
**Pull request by LuisGuenther**
# Updating documentation of user controls

Issue: #3087 

## Description

**Documentation (screenshot) for user controls updated to also include custom views and suspicious metrics.**

### CODE REVIEW

**Review by ce-bo — COMMENTED** [review]
Would you please add the new "link metrics" symbol? Thank you!

**Comment by ce-bo** [comment]
In the Custom View documentation, there is the following text
`Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a "shortcut-button". In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
`

I would suggest to update the term "shortcut-button" to "quick add button" and link it to the user-controls docs. Therefore, we need to separate the to Custom Views buttons in the user-controls-screenshot.

**Comment by LuisGuenther** [comment]
Thank you for your feedback @ce-bo

**Comment by LuisGuenther** [comment]
I added a header to indicate the enabled experimental features. I can later be removed by clipping the image when the features are officially made stable.
I also added explanations for the "link metrics" and the "custom view shortcut" buttons.

File: gh-pages/_docs/06-04-custom-view.md [hunk 1]
```diff
@@ -3,8 +3,10 @@ permalink: /docs/custom-view/
 title: "Custom View"
 ---
 
-Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a "shortcut-button". In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
+Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a [quick add button]({{site.baseurl}}{% link _docs/06-02-user-controls.md %}) in the menu bar. In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
```
Comment (ce-bo) [inline gh-pages/_docs/06-04-custom-view.md:6]:
```suggestion
Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a [quick add button]({{site.baseurl}}{% link _docs/06-02-user-controls.md %}) in the menu bar. In order to be able to access more settings, there is another button next to the "quick add button" with the following functions:
```

**Pull request merged**